assets/src/block-validation/components/error/get-error-source-title.js

```diff
@@ -86,7 +90,7 @@ export function getErrorSourceTitle( sources = [] ) {
 		output.push( __( 'Core', 'amp' ) );
 	}
 
-	if ( 0 === output.length && 0 < sources.length ) {
+	if ( ! output.length && ! sources?.length ) {
```

Comment:
I think my previous suggestion was wrong. It seems this should rather be:

```suggestion
	if ( ! output.length && Array.isArray( sources ) && 0 < sources.length ) {
```

Reply:
Hmm, yeah, the check should be inverted - we're looking for a case when the `output` is empty and there is at least one `source`. Right now we check if there are no sources.

I think, though, that we could keep the optional chaining and just remove the negation operator like:

```suggestion
	if ( ! output.length && sources?.length ) {
```

Check out this example:

![Screenshot 2022-01-21 at 12 40 19](https://user-images.githubusercontent.com/478735/150521129-2d151a6b-7a44-4866-b626-7abd406b5c65.png)